book.toml

```diff
@@ -29,6 +29,18 @@ verbose = false # Report timing information.
 # [preprocessor.aspect-ratio-helper]
 # command = "./aspect-ratio-helper.py"
 
+[output.pandoc]
+optional = true
+
+[output.pandoc.profile.pdf]
+output = "comprehensive-rust.pdf"
+pdf-engine = "lualatex"
+
+[output.pandoc.profile.pdf.variables]
+geometry = ["margin=1.25in"]
+linkcolor = "blue"
+urlcolor = "red"
+
```

Comment:
Just so we have it here: I would like to try using the Nota fonts by default since they seem to do what they say:

```toml
[output.pandoc.profile.pdf.variables]
mainfont = "Noto Serif"
sansfont = "Noto Sans"
monofont = "Noto Sans Mono"

CJKmainfont = "Noto Serif CJK SC"
CJKsansfont = "Noto Sans CJK SC"
CJKmonofont = "Noto Sans Mono CJK SC"
```

You would need to more logic in `publish.yml` to either set or clear the `CJK*` variables for non-CJK languages.

Reply:
Worked around the inability to set CJK* with env vars by passing Pandoc a language specific file with extra options (`pandoc/zh-CN.yaml`, etc.). [Rendered](https://max-heller.com/comprehensive-rust/zh-CN/comprehensive-rust.pdf)

Reply:
Wow, very nicely done!